spec/requests/create_resource_spec.rb

```diff
@@ -141,9 +141,10 @@
       file_params['size'] = 10
       model_params
     end
+    let(:path) { '/v1/resources' }
```

Comment:
I find that putting this as a variable is harder to read than how it was before.  This isn't used more than one time, so why?

Reply:
was cruft from a slightly different implementation, moved back inline